airbyte-server/build.gradle

```diff
@@ -1,28 +1,5 @@
 plugins {
     id 'application'
-    id 'maven-publish'
-    id 'com.github.johnrengelman.shadow' version '6.1.0'
-}
-
-shadowJar {
-    zip64 true
-    mergeServiceFiles()
-    exclude 'META-INF/*.RSA'
-    exclude 'META-INF/*.SF'
-    exclude 'META-INF/*.DSA'
-    // Not stubbing this out adds 'all' to the end of the jar's name.
-    classifier = ''
-}
-
-publishing {
-    publications {
-        shadow(MavenPublication) { publication ->
-            project.shadow.component(publication)
-        }
-    }
-    repositories {
-        mavenLocal()
```

Comment:
we still need to publish locally for dev/testing

Reply:
should be ok now